docs/data/charts/pyramid-demo/pyramid.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Charts - Pyramid demonstration
+productId: x-charts
+components: FunnelChart, FunnelPlot
+---
+
+# Charts - Pyramid demonstration
```

Comment:
Should we just call it Pyramid chart? I had the impression we were going to remove demo pages from our docs and try to document everything in a single page. 

Since the pyramid chart is a different chart, it makes sense to have it as a separate page, but I'd treat it just like a normal chart. What do you think?